src/components/CytoscapeGraph/CytoscapeGraph.tsx

```diff
@@ -206,57 +204,66 @@ export class CytoscapeGraph extends React.Component<CytoscapeGraphProps, Cytosca
 
     this.trafficRenderer.stop();
 
+    const isTheGraphSelectd = this.cy.$(':selected').length === 0;
```

Comment:
I hate doing this but... missing the last "e" in the variable name ("Selected" not "Selectd").

Reply:
will fix, thanks